cmd/mungedocs/mungedocs.go

```diff
@@ -34,30 +34,32 @@ var (
 
 	ErrChangesNeeded = errors.New("mungedocs: changes required")
 
+	// All of the munge operations to perform.
 	// TODO: allow selection from command line. (e.g., just check links in the examples directory.)
-	mungesToMake = munges{
-		munger(updateTOC),
-		munger(checkLinks),
+	allMunges = []munge{
+		{"table-of-contents", updateTOC},
+		{"check-links", checkLinks},
```

Comment:
adding names like this and then a flag to select which to run was my next step, hehe.